src/main/java/com/bulletinboard/post/controller/PostController.java

```diff
@@ -0,0 +1,57 @@
+package com.bulletinboard.post.controller;
+
+import com.bulletinboard.post.dto.PostNewRequest;
+import com.bulletinboard.post.dto.PostResponse;
+import com.bulletinboard.post.dto.PostUpdateRequest;
+import com.bulletinboard.post.dto.PostUpdateResponse;
+import com.bulletinboard.post.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class PostController {
```

Comment:
```suggestion
@RequiredArgsConstructor
@RequestMapping("posts")
public class PostController {
```

이렇게 prefix를 넣어주면 하위 메서드에서 posts를 생략할 수 있어서 좋을 것 같아요

Reply:
https://github.com/JSCODE-EDU/project-class-JeongheonHa/blob/8376191af7f5049dfbba6af0a5cca967b56553be/src/main/java/com/bulletinboard/post/controller/PostController.java#L20

경로를 통합하니까 확실히 더 가독성이 좋아진 것 같아요.